visualization/app/codeCharta/ui/codeMap/threeViewer/threeOrbitControlsService.ts

```diff
@@ -56,30 +103,52 @@ export class ThreeOrbitControlsService {
 		this.threeCameraService.camera.translateZ(oldZoom)
 	}
 
-	public autoFitTo(obj = this.threeSceneService.mapGeometry) {
-		const boundingSphere = new THREE.Box3().setFromObject(obj).getBoundingSphere()
+	private setDefaultZoom() {
+		this.defaultZoom = this.getZoom()
+	}
 
+	private autoFitToCalculation(boundingSphere) {
 		const cameraReference = this.threeCameraService.camera
 
 		const scale = 1.4 // object size / display size
 		const objectAngularSize = ((cameraReference.fov * Math.PI) / 180) * scale
 		const distanceToCamera = boundingSphere.radius / Math.tan(objectAngularSize / 2)
 		const len = Math.sqrt(Math.pow(distanceToCamera, 2) + Math.pow(distanceToCamera, 2))
 
-		cameraReference.position.set(len, len, len)
-		this.controls.update()
+		return len
+	}
 
+	private focusCameraViewToCenter(boundingSphere) {
 		const t: Vector3 = boundingSphere.center.clone()
 		t.setY(0)
 		this.controls.target.set(t.x, t.y, t.z)
 
-		cameraReference.lookAt(t)
-
-		this.defaultCameraPosition = cameraReference.clone().position
+		this.threeCameraService.camera.lookAt(t)
 
 		this.threeCameraService.camera.updateProjectionMatrix()
 	}
 
+	private initializeDefaultZoomWithoutAutoFit() {
+		const autoFittedPerspective: Vector3 = new Vector3(0, 0, 0)
+		const boundingSphere = this.getBoundingSphere()
+		const len: number = this.autoFitToCalculation(boundingSphere)
```

Comment:
What the function does is to calculate some sort of length, maybe use a descriptive name of what is actually calculated here as method name instead of AutoFitToCalculation?

Reply:
Changed it to a more descriptive name :)